docs/user/custom_worker.md

```diff
@@ -0,0 +1,149 @@
+# Change how your algorithm samples (Implement a Custom Worker)
+
+```eval_rst
+In sampling, :code:`Worker` perform rollouts to get samples. In this tutorial,
+we will implement a custom worker to meet the requirements of the RL2
+algorithm :cite:`duan2016rl`.
+```
+
+## `Worker` interface and `DefaultWorker`
+
+In Garage, all `Worker`s need to implement the [`Worker` interface](https://garage.readthedocs.io/en/latest/_autoapi/garage/sampler/index.html#garage.sampler.Worker).
+Garage provides a [`DefaultWorker`](https://garage.readthedocs.io/en/latest/_autoapi/garage/sampler/index.html#garage.sampler.DefaultWorker)
+for general algorithms. Below is the `rollout()` method of `DefaultWorker`, the
+most important method of `Worker`.
+
+```py
+    def rollout(self):
+        """Sample a single rollout of the agent in the environment.
+
+        Returns:
+            garage.TrajectoryBatch: The collected trajectory.
+
+        """
+        self.start_rollout()
+        while not self.step_rollout():
+            pass
+        return self.collect_rollout()
+```
+
+To implement a custom worker, an easy way is to override the methods we want to
+change (e.g. `rollout()`) instead of implementing our custom worker from
+scratch. For example:
+
+```py
+from garage.sampler import DefaultWorker
+
+class MyWorker(DefaultWorker):
+    """Initialize a worker.
+
+    Args:
+        seed(int): The seed to use to initialize random number generators.
+        max_path_length(int or float): The maximum length paths which will
+            be sampled. Can be (floating point) infinity.
+        worker_number(int): The number of the worker where this update is
+            occurring. This argument is used to set a different seed for each
+            worker.
+
+    Attributes:
+        agent(Policy or None): The worker's agent.
+        env(gym.Env or None): The worker's environment.
+
+    """
+    def __init__(
+            self,
+            *,  # Require passing by keyword, since everything's an int.
+            seed,
+            max_path_length,
+            worker_number):
+        ...
+        super().__init__(seed=seed,
+                         max_path_length=max_path_length,
+                         worker_number=worker_number)
+        ...
+
+    def rollout(self):
+        ...
+```
+
+## Custom Worker for RL2
```

Comment:
at the end, can you show us how this custom worker is going to be used in a launcher?